src/types/task.ts

```diff
@@ -100,3 +101,11 @@ export enum TaskStatus {
   UndispatchedUmbrella = "undispatched-umbrella",
   ScheduledUmbrella = "scheduled-umbrella",
 }
+
+export enum CommitType {
+  Base = "base",
+  LastPassing = "lastPassing",
+  LastExecuted = "lastExecuted",
+}
+
+export type CommitTask = GetBaseVersionAndTaskQuery["task"]["baseTask"];
```

Comment:
These might make sense in a `task/actionButtons/types.ts` (or something similar) file instead. These are very specific to that component and would likely be better off living colocated where they are used.